packages/formik/test/FormikContext.test.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { render } from 'react-testing-library';
+import { Formik } from '../src/Formik';
+import { useFormikContext } from '../src/FormikContext';
+
+describe('FormikContext', () => {
+  describe('useFormikContext', () => {
+    it('should return validationContext if set', () => {
+      const validationSchema = 'validationSchema';
```

Comment:
is `string` valid for validation schema? thought it had to be a Yup object. Other tests use `{ validate: jest.fn(() => Promise.resolve()) }`.